contracts/BaseStrategy.sol

```diff
@@ -131,20 +131,27 @@ abstract contract BaseStrategy {
     IERC20 public want;
 
     // So indexers can keep track of this
-    event Harvested(uint256 wantEarned, uint256 lifetimeEarned);
+    event Harvested(uint256 profit);
```

Comment:
mmm should we do the `losses` change in this version?

Reply:
That's a bigger change I hadn't thought about yet. Accepting suggestions though